_posts/2020-08-19-runge20a.md

```diff
@@ -27,7 +27,7 @@ lastpage: 120
 page: 110-120
 order: 110
 cycles: false
-bibtex_author: Runge, Jakob and Tibau, Xavier-Andoni and Bruhns, Matthias and Mu{\~~n}oz-Mar\'i,
+bibtex_author: Runge, Jakob and Tibau, Xavier-Andoni and Bruhns, Matthias and Muñoz-Marí,
```

Comment:
The bibtex_authors field should be in LaTeX form, could you replace with Mu\~{n}oz-Mar\'{i} and resubmit the pull request? Thanks!